src/Symfony/Component/Workflow/Definition.php

```diff
@@ -51,7 +52,21 @@ public function __construct(array $places, array $transitions, string $initialPl
      */
     public function getInitialPlace()
     {
-        return $this->initialPlace;
+        @trigger_error(sprintf('Calling %s::getInitialPlace is deprecated. Call %s::getInitialPlaces instead.', __CLASS__, __CLASS__));
```

Comment:
missing corresponding annotation
do we want to deprecate passing a string as argument to the constructor?

Reply:
> missing corresponding annotation
Fixed.
> do we want to deprecate passing a string as argument to the constructor?
Do not worth it